cmd/scheduler/main.go

```diff
@@ -43,7 +43,8 @@ func main() {
 	if viper.GetBool(MigrateDatabase) {
 		migrateDatabase(&config)
 	} else {
-		scheduler.Run(&config)
+		err := scheduler.Run(&config)
+		panic(errors.WithMessage(err, "Failed to run scheduler"))
```

Comment:
panic() will change behavior -- I think prefer "log it" approach used elsewhere for this lint fixup PR